src/ecmult_multi_impl.h

```diff
@@ -0,0 +1,112 @@
+/**********************************************************************
```

Comment:
Having this in a separate file from ecmult_impl.h I think will result in a circular dependency between strauss and the dispatch code.